packages/ast-client/src/query.test-d.ts

```diff
@@ -38,65 +38,65 @@ const emptyRecordQuery = {} as Query<Record<string, never>, MyQueryResult>;
 
 describe("Queries", () => {
   test("are not plain document nodes", () => {
-    expect(myQuery).type.not.toEqual<DocumentNode>();
-    expect(voidQuery).type.not.toEqual<DocumentNode>();
-    expect(undefinedQuery).type.not.toEqual<DocumentNode>();
-    expect(emptyQuery).type.not.toEqual<DocumentNode>();
-    expect(emptyRecordQuery).type.not.toEqual<DocumentNode>();
+    expect(myQuery).type.not.toBe<DocumentNode>();
+    expect(voidQuery).type.not.toBe<DocumentNode>();
+    expect(undefinedQuery).type.not.toBe<DocumentNode>();
+    expect(emptyQuery).type.not.toBe<DocumentNode>();
+    expect(emptyRecordQuery).type.not.toBe<DocumentNode>();
   });
 
   test("All queries are DocumentNodes", () => {
-    expect<DocumentNode>().type.toBeAssignable(myQuery);
-    expect<DocumentNode>().type.toBeAssignable(voidQuery);
-    expect<DocumentNode>().type.toBeAssignable(undefinedQuery);
-    expect<DocumentNode>().type.toBeAssignable(emptyQuery);
-    expect<DocumentNode>().type.toBeAssignable(emptyRecordQuery);
+    expect(myQuery).type.toBeAssignableTo<DocumentNode>();
+    expect(voidQuery).type.toBeAssignableTo<DocumentNode>();
+    expect(undefinedQuery).type.toBeAssignableTo<DocumentNode>();
+    expect(emptyQuery).type.toBeAssignableTo<DocumentNode>();
+    expect(emptyRecordQuery).type.toBeAssignableTo<DocumentNode>();
```

Comment:
Source and target got swapped in these assertions. The new `.toBeAssignableTo()` matcher now allows keeping the target type on right side similar to other cases.

I understood that `.toBeAssignableTo()` is missing while looking through your repository. Thanks for using TSTyche (;